src/components/App.tsx

```diff
@@ -47,8 +49,15 @@ export default class App extends Component<AppProps, AppState> {
         this.setState(newState);
     }
     public render(): ReactNode {
+        const { alertBannerVisible, } = this.state;
         return (
             <div id="app">
+                <div className="alert-banner" style={alertBannerVisible ? {} : { display: "none", }}>
+                    <span className="close-ab" onClick={this.handleClickCloseAlertBanner.bind(this)}>&times;</span>
+                    Interested in contributing or suggesting ideas and splits? Check out the &nbsp;
```

Comment:
Nitpick: I _think_ what you want is
```suggestion
                    Interested in contributing or suggesting ideas and splits? Check out the&nbsp;
```